Tryouts/Visuals/Windows/WindowsVisualUtils/JSCommunicationsClient.cs

```diff
@@ -0,0 +1,89 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using Google.Protobuf;
+using Microsoft.Web.WebView2.WinForms;
+using MorganStanley.ComposeUI.Tryouts.Core.Abstractions;
+using Subscriptions;
+using System.Reactive.Linq;
+using System.Runtime.InteropServices;
+using System.Text.Json;
+
+namespace MorganStanley.ComposeUI.Prototypes.WV2Prototype
+{
+    [ClassInterface(ClassInterfaceType.AutoDual)]
+    [ComVisible(true)]
+    public class JSCommunicationsClient 
```

Comment:
This is a 'host object' for WebView2: https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.addhostobjecttoscript?view=webview2-dotnet-1.0.1185.39

Reply:
It is Window Specific because currently WebView2 works only for Windows. They plan to make it multiplatform (and I hope they'll follow up on their promise) but not yet. 

AddHostObjectToScript method adds C# objects to become available from within the HTML scripts which I need in order to wire the communications.

Reply:
Got it, thanks for the explanation